UIforETW/Utility.cpp

```diff
@@ -274,22 +275,31 @@ std::wstring StripExtensionFromPath(const std::wstring& path)
 int DeleteOneFile(HWND hwnd, const std::wstring& path)
 {
 	std::vector<std::wstring> paths;
-	paths.push_back(path);
+	paths.emplace_back(path);
 	return DeleteFiles(hwnd, paths);
 }
 
 int DeleteFiles(HWND hwnd, const std::vector<std::wstring>& paths)
 {
+	ATLASSERT(paths.size() > 0);
+
+	size_t totalLength = 1;

```

Comment:
Not worth it. Yes, reserve is a good idea, but in this program it will *never* make a noticeable difference (vector is still O(N) even without reserve). So it is definitely not worth a loop to precalculate the required size.

I think that in UIforETW calling reserve() is mostly pretentious, and here it is definitely excessive. Less code is important also.